lighthouse-core/test/lib/traces/tracing-processor-test.js

```diff
@@ -159,4 +159,62 @@ describe('TracingProcessor lib', () => {
       assert.deepEqual(results, expected);
     });
   });
+
+  describe('log normal distribution', () => {
+    it('creates a log normal distribution', () => {
+      // This curve plotted with the below percentile assertions
+      // https://www.desmos.com/calculator/vjk2rwd17y
+
+      const median = 5000;
+      const pODM = 3500;
+      const distribution = TracingProcessor.getLogNormalDistribution(median, pODM);
+
+      function getPct(distribution, value) {
+        return distribution.computeComplementaryPercentile(value).toFixed(2);
```

Comment:
these string comparisons are somewhat horrifying :) Somewhat just jumping through hoops, but maybe `return Number(distribution.computeComplementaryPercentile(value).toFixed(2))`? and use numbers below?